### ISSUE TITLE
Error running docker containers for 8.0 and 8.1 stable images on ARM64 architecture.

### ISSUE THREAD

**Issue by rodrigo-lourenco-lopes** [camunda/zeebe#13768]
**Describe the bug**

The dockerfile for 8.0 and 8.1 do not support ARM64 architectures. 
This makes it impossible to run tests locally that build docker images.

Error shown within the docker container zeebe:current-test:
`exec /usr/local/zeebe/bin/tini: no such file or directory`

The support for ARM64 was only introduced in 8.2. The difference being that tini is being downloaded as a raw x86 binary instead from the apt repositories.

**To Reproduce**

For example, run QA tests on 8.0 or 8.1 on a computer with ARM64.

**Suggested Solution**

Backport only the changes related to Tini if no issue is caused.

**Comment by megglos** [comment]
IIRC rocksDb only supported arm with a later release, I'm not sure if we depend on those already with 8.0/8.1 @npepinpe do you remember?

**Comment by npepinpe** [comment]
AFAIK 8.0 and 8.1 never supported ARM64. We introduced that in 8.2. And some of it was related to RocksDB, but I don't remember all the changes that went into ARM64 support. Might've been others.

**Comment by megglos** [comment]
@rodrigo-lourenco-lopes if you wanna run AMD64 images locally on arm macs, I can recommend using https://github.com/abiosoft/colima#customizing-the-vm it now uses rosetta for am64 emulation and the performance was acceptable the last time I used it. (hugely better than qemu)

**Comment by rodrigo-lourenco-lopes** [comment]
I also been just using the latest docker file in my machine whenever I needed to run these tests. 

I created a test branch with the changes necessary for the support of aarch64 in stable/8.1, and although this is useful locally, other changes would be necessary to pass the docker tests.

Should we then close this issue since there are several alternatives to run these tests locally?

https://github.com/camunda/zeebe/pull/13770/commits/14bbb5ee1d16145895685370a498be649c3104a5
